setup-scripts/support/authorized_keys

```diff
@@ -6,3 +6,4 @@ ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIAyIT2HqzDhQs6jS89ZsnY6+GJEklVMqF6fXe/i5s8d7
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIFxLuZzjmFN9CgVOI5vaiVhQgMwG9dLQJ688wrsbpHH/ ivaylo@nebulous.tech
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAINbAhwjJNAud7YIJvLth2bmeUg3kO20xl7ZfqBTvoXn8 Filip Rysavy
 ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIG67M3zC4eDJEjma0iKKksGclteKbB86ONQtBaWY93M6 mjsevey@gmail.com
+ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIN6Kcx8yetova4/ALUQHigo/PBMJO33ZTKOsg2jxSO2a user@depl
```

Comment:
can `user@depl` be a bit more meaningful, maybe we could alias the machine to deploy.siasky.net and have it say user@deploy.siasky.net ?

Reply:
Changed the hostname to `depl.siasky.dev`,
edited the key on the `depl` server
and updated the user to `user@depl.siasky.dev` in this repo.

Just FYI: I have tested it and the user string can be arbitrary,
it is ignored by SSH,
they do not have to match between client and server.